### ISSUE TITLE
The prior value added twice in ThrowerMarkHarz toy generator

### ISSUE THREAD

**Issue by ulyevarou** [gundam-organization/gundam#526]
In `src/Utils/src/ParameterThrowerMarkHarz.cpp` the addition of `(*pvals)(i)` should be removed

```
@@ -54,7 +54,8 @@ void ParameterThrowerMarkHarz::ThrowSet(std::vector<double> &parms)
   }
   TVectorD prod = (*chel_dec) * std_rand;
   for (int i = 0; i < npars; i++)
-    parms[i] = prod(i) + (*pvals)(i);
+    parms[i] = prod(i);
 }
```

since it is added later in the 
```
void ParameterSet::throwParameters(..) ->
par.setThrowValue( par.getPriorValue() + gain_ * throwsList[iPar] );
```

**Comment by ClarkMcGrew** [comment]
I've applied my understanding of the fix in #557.  @ulyevarou Can you check that I correctly understood your suggested fix and did it correctly?  If I did it correctly, feel free to merge and close this, or let me know and I will.

**Comment by ClarkMcGrew** [comment]
Fixed by #557
